module/config.mjs

```diff
@@ -3234,7 +3262,7 @@ function patchConfig(key, fallbackKey, options) {
 
   Object.values(DND5E[key]).forEach(o => {
     if ( foundry.utils.getType(o) !== "Object" ) return;
-    o.toString = toString;
+    Object.defineProperty(o, "toString", {value: toString});
```

Comment:
Adding `toString` as an non-enumerable property. Otherwise `toString` appears in `CONFG.statusEffects` entries.